packages/devtools_app/lib/src/flutter/flex_split_column.dart

```diff
@@ -0,0 +1,132 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+import 'package:flutter/material.dart';
+
+import 'split.dart';
+
+class FlexSplitColumn extends StatelessWidget {
+  FlexSplitColumn({
+    Key key,
+    @required this.totalHeight,
```

Comment:
instead of passing in the height let users wrap the FlexSplitColumn in the widget they want to give it the size they want.

Reply:
we need totalHeight for the calculation done in the static method `adjustInitialFractions`